modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -376,6 +376,343 @@ Additionally, the user requires some `ReadOnly` permissions, and some of the rol
 
 For more information about creating an account with only the required privileges, see link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-5372F580-5C23-4E9C-8A4E-EF1B4DD9033E.html[vSphere Permissions and User Management Tasks] in the vSphere documentation.
 
+
+[discrete]
+[id="installation-vsphere-installer-infra--minimum-requirements_{context}"]
+== Minimum required vCenter account privileges
+
+You can create a custom role with privilege sets that propogate a minimum set of permissions to each vSphere object. Depending on the installation stage, you might need to restrict access to certain vSphere objects. 
```

Comment:
A user can create a custom role through the vCenter Client or is some sort of CLI option available?

https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-security/GUID-18071E9A-EED1-4968-8D51-E0B4F526FDA3.html

Reply:
I see a note in this link that states "When you add a custom role and do not assign any privileges to it, the role is created as a Read Only", so I should state this, because the cloud provider minimum privileges is based around this default"?

Reply:
Thanls, @rvanderp3 . Updated. I'll move the PR to the next review cycle.